### ISSUE TITLE
ByteTrack with detections from text files

### ISSUE THREAD

**Issue by hlmhlr** [ifzhang/ByteTrack#161]
Hi,

I have detections in .txt file. Is there option to use this repository by importing the detections from a text files without using the yolox detector?
Thanks in advance

**Comment by LamnouarMohamed** [comment]
yes, I think no option in this repository but you need to code a little bit. (take care about range of you number in you file because inside byteTrack there is some change of scale ok)

**Comment by noahcao** [comment]
You can do this by using the detection read from existing txt files instead of from the detector. I don't the current ByteTrack supports this but that would be trivial to adapt. This is [an example](https://github.com/noahcao/OC_SORT/blob/b79a141d3e9e682075432ad215538961ee2b0a24/trackers/ocsort_tracker/ocsort.py#L327) implementation from another MOT repo OC-SORT. You can do the same thing within ByteTracker implementation.
